learning_resources/serializers.py

```diff
@@ -479,11 +478,11 @@ class ContentFileSerializer(serializers.ModelSerializer):
     resource_readable_num = serializers.CharField(
         source="run.learning_resource.resource_num"
     )
-    resource_type = serializers.SerializerMethodField()
+    file_content_type = serializers.SerializerMethodField()
```

Comment:
This refers to the "Resource Type" in the sidebar of the ocw search
https://ocw.mit.edu/search/?type=resourcefile

Examples are "Lecture Notes" , "Assignments. To avoid confusion with the resource type form Learning Resources ("course, "program") i renamed this to file_content_type. But maybe that's not great since we also have content_type and file_type fields. Any other name suggestions?

Reply:
resource_category sounds great

Reply:
Or actually content_category is better